server/models/userCourseEnrollment.js

```diff
@@ -0,0 +1,49 @@
+const { DataTypes } = require('sequelize');
+const sequelize = require('../config/db.config').sequelize;
+const User = require('./user');
+const Course = require('./course');
+
+const UserCourseEnrollment = sequelize.define('user_course_enrollment', {
+    id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+    },
+
+    enrollment_date: {
+        type: DataTypes.DATE,
+        allowNull: false
+    },
+
+    enrollment_status: {
```

Comment:
Why not use status instead as the column name?

Reply:
Changed column name from enrollment_status to status.